pig/src/org/partiql/pig/cmdline/Command.kt

```diff
@@ -20,5 +20,5 @@ import java.io.File
 sealed class Command {
     object ShowHelp : Command()
     data class InvalidCommandLineArguments(val message: String) : Command()
-    data class Generate(val typeUniverseFile: File, val outputFile: File, val target: TargetLanguage) : Command()
+    data class Generate(val typeUniverseFile: File, val target: TargetLanguage) : Command()
```

Comment:
This change along with the change in `TargetLanguage` would appear to break clients using the existing API such as PartiQL ([build.gradle](https://github.com/partiql/partiql-lang-kotlin/blob/c6e3fc651b/lang/build.gradle#L70-L73) uses Kotlin as the target language with the "outputFile" argument).

1. Would this change result in a new major version for PIG?
2. Should we add some backwards compatibility option (e.g. allow client code to use the old API to specify one target file)?

Reply:
1. Without a backward compatibility option, yes.
2. I would say that we don't have a large enough number of customers to justify the additional effort in providing a backward compatibility option.

Reply:
Ok, I think as long as it's a new major version and it's simple + clear how to migrate, a backward compatibility option isn't required.